src/js/owl.navigation.js

```diff
@@ -187,7 +187,7 @@
 		this._controls.$absolute = (settings.dotsContainer ? $(settings.dotsContainer)
 			: $('<div>').addClass(settings.dotsClass).appendTo(this.$element)).addClass('disabled');
 
-		this._controls.$absolute.on('click', 'button', $.proxy(function(e) {
+		this._controls.$absolute.on('click', '.' + settings.dotClass, $.proxy(function(e) {
```

Comment:
This might cause unpredictable behaviour, when you are not using custom dots.

Reply:
Thanks, it help me!

Reply:
thanks, you saved my life :-)